### DISCUSSION TITLE
Should focus ring stay on a combobox when navigating between options?

### DISCUSSION THREAD

**Discussion by tomaszferens** [adobe/react-spectrum#4162] [General]
Compare this example from w3.org

https://user-images.githubusercontent.com/17815179/223167477-71be619d-3f36-409d-90a4-bec8dd77c9cc.mov

source: [codepen](https://codepen.io/pen?&prefill_data_id=068b2a5f-4433-4488-be84-cce1b23b4ff6)

with react-aria `useComboBox` hook:

https://user-images.githubusercontent.com/17815179/223167555-44e24c9a-f74f-4aa2-b283-82fc423822d2.mov

In the w3.org example, the focus ring disappears from a combobox once you start selecting options (arrow down/up on keyboard) and then it returns back as you continue typing. When using `useComboBox` hook, the focus ring is always visible.

Which behaviour is the "correct" one or are both valid?

**Comment by LFDanLu** [comment]
I'm not entirely sure, https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-autocomplete-both/ doesn't explicitly mention a hard requirement for the focus ring on the combobox input when virtual focus is within the list. I can see an argument for either, depending on if it is more confusing to have two elements on the page with a focus ring like style (i.e the input and the focused element in the list) or if it is important to visually show that the combobox input is still focused even when navigating the options in the list. 

@majornista @jnurthen any opinions on the above?
